opencog/cogutil/Dockerfile

```diff
@@ -4,12 +4,6 @@
 
 FROM opencog/opencog-deps
 
-# Install repositories and dependencies
-# FIXME: when octools auto-updates remove this.
-ADD https://raw.githubusercontent.com/opencog/ocpkg/master/ocpkg \
-    /tmp/octool
-RUN chmod 755 /tmp/octool
-
```

Comment:
The reason for this was to speed up build of new images when there is change in octool. Why are you removing it?

Reply:
Because it allows installing dependencies and `cogutil` using different `octool` versions which can be incompatible.

The immediate problem to me is that in `singnet` repo I pass specific `octool` version into docker building procedure to build docker from any branch not only from master. And recently I found that `cogutil` uses hardcoded `master` version. I see two solutions:
- pass `octool` version as parameter for `cogutil` as well and made this change in `singnet` repo;
- remove getting `octool` version from `cogutil` image Dockerfile.

I chose second option as it looks logical for me. But I can implement first one as well.